collect_app/src/main/java/org/odk/collect/android/tasks/DeleteInstancesTask.java

```diff
@@ -63,10 +62,12 @@ protected Integer doInBackground(Long... params) {
                 deleted += wasDeleted;
 
                 if (wasDeleted > 0) {
-                    Collect.getInstance().getActivityLogger().logAction(this, "delete", deleteForm.toString());
+                    Collect.getInstance()
+                            .getActivityLogger().logAction(this, "delete", deleteForm.toString());
                 }
             } catch (Exception ex) {
-                Log.e(TAG, "Exception during delete of: " + param.toString() + " exception: " + ex.toString());
+                Timber.e(ex, "Exception during delete of: "
```

Comment:
Example of code that throw log trace to the black hole 🤣 

Not your fault @shobhitagarwal1612 though. And I think we will have separate PR to fix this kind of issue 👍